Katello/katello
# Fixes #36411 - add ability to skip treeinfo files

#### What are the changes introduced in this pull request?
- Adds a new checkbox to skip treeinfo files
- Rewrites a treeinfo Pulp error to give better workaround instructions

#### Considerations taken when implementing this change?

#### What are the testing steps for this pull request?
1) Sync https://www.keepersecurity.com/kcm/2/el/8/x86_64
2) Verify the error in the Dynflow action
3) Enable treeinfo skipping
4) Sync again
5) Verify that the sync is successful
